interface/main/calendar/modules/PostCalendar/pntemplates/default/views/header.html

```diff
@@ -1,6 +1,8 @@
 <html>
 <head>
+[-php-]require_once "{$GLOBALS['srcdir']}/templates/standard_header_template.php";[-/php-]
```

Comment:
I think this is the right spot for the calendar (this should now work with the new fixed standard_header_template.php just added to the codebase). And you should be able to remove this line in the above calendar script and also remove lines 5-7 and 17-19
(hopefully the new jquery will jive with stuff and that calendarDirectSelect.js thing)